Extend Log RBAC with Logs Pipelines, Indexes and Archives best practices. (#8698)
DataDog/documentation: content/en/account_management/rbac/permissions.md

Grants the ability to write historical views, meaning to trigger a [Log Rehydration*][11].

> This permission is global, but only enables to trigger a rehydration for Archives users have [Logs Read Archive](r#logs-read-archives) permission.

Reviewer: ```suggestion
This permission is global. It enables users to trigger a rehydration for archives on which they have [Logs Read Archive](r#logs-read-archives) permission.
```

{{< img src="account_management/rbac/logs_hv_roles_combination.png" alt="Write Historical View"  style="width:70%;">}}

> On the example above:

Reviewer: ```suggestion
In the example above:
```

> * `ADMIN` Role members **can** rehydrate from the `Audit Archive`, as they have the Write Historical View (Rehydrate) permission on along with the Read Archive permission on that Archive.

Reviewer: ```suggestion
* `ADMIN` Role members **can** rehydrate from the `Audit Archive`, as they have the Write Historical View (Rehydrate) permission, as well as the Read Archive permission on that archive.
```

> * `AUDIT` Role members **cannot** rehydrate from the `Audit Archive`, as they don't have the Write Historical View (Rehydrate) permission on.

Reviewer: ```suggestion
* `AUDIT` Role members **cannot** rehydrate from the `Audit Archive`, as they do not have the Write Historical View (Rehydrate) permission.
```

> * `PROD` Role members **cannot** rehydrate from the `Audit Archive`, as they don't have Read Archive permission on that Archive.

Reviewer: ```suggestion
* `PROD` Role members **cannot** rehydrate from the `Audit Archive`, as they do not have the Read Archive permission on that archive.
```
